docling_core/types/doc/document.py

```diff
@@ -57,6 +68,9 @@
 from docling_core.types.doc.tokens import _LOC_PREFIX, DocumentToken, TableToken
 from docling_core.types.doc.utils import relative_path
 
+if TYPE_CHECKING:
+    from docling_core.transforms.visualizer.base import BaseVisualizer
```

Comment:
Perhaps this can be done like for the other visualizer types, i.e. within `get_visualization()`.